AuthenticatorBridgeKit/FetchedResultsPublisher.swift

```diff
@@ -0,0 +1,145 @@
+import Combine
+import CoreData
+
+// MARK: - FetchedResultsPublisher
+
+/// A Combine publisher that publishes the initial result set and any future data changes for a
+/// Core Data fetch request.
+///
+/// Adapted from https://gist.github.com/darrarski/28d2f5a28ef2c5669d199069c30d3d52
+///
+class FetchedResultsPublisher<ResultType>: Publisher where ResultType: NSFetchRequestResult {
```

Comment:
🤔 Does it make sense to have tests wrapping this, just in case?

Reply:
This was just a pickup form BitwardenShared and I didn't see any (direct) tests there. I can try adding some if we think it should be tested directly. It does seem to be tested as part of other tests however.

Reply:
We're probably good, just wanted to double-check.